docs/plans/20260120_deployment_architecture.md

```
@@ -0,0 +1,90 @@
+│       └── .env.data.example     # 데이터용 템플릿
+│
+├── ...
+```
+
+## 3. 계층별 서비스 및 설정 상세
+
+### 3.1. 데이터 계층 (Data Layer) - EC2 Instance B
+*안정적인 인프라스트럭처. CI/CD 배포 대상에서 제외(수동/별도 관리).*
+
+*   **포트 개방 (Inbound)**: App Instance의 Private IP에 대해서만 허용.
+    *   `5432` (Postgres), `6379` (Redis), `9200` (OpenSearch), `9100` (Node Exp), `9187` (Pg Exp), `9121` (Redis Exp).
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)